install/kubernetes/templates/istio-auth/istio-cluster-ca.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       serviceAccountName: istio-ca-service-account
       containers:
       - name: istio-ca
-        image: docker.io/istio/istio-ca:0.1.5
+        image: {CA_HUB}/istio-ca:{CA_TAG}
```

Comment:
I think it would be cleaner to use go template. Which would simplify the go code anyway.